src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -36,10 +37,11 @@ trait CromIamApiService extends Directives with SprayJsonSupport with DefaultJso
   implicit val materializer: ActorMaterializer
 
   protected def configuration: CromIamServerConfig
-  protected lazy val UserIdHeader: String = configuration.cromIamConfig.userIdHeader
 
-  override protected lazy val samActor: ActorRef = system.actorOf(SamActor.props(configuration.cromIamConfig.userIdHeader, configuration.cromIamConfig.allowedUsers), "SamActor")
-  override protected lazy val actorRefFactory: ActorRefFactory = system
+  System.setProperty("jsse.enableSNIExtension", "false")
```

Comment:
Without this, if Cromwell is on an SSL connection we get `womp womp! cromiam.webservice.CromIamApiService$CromwellConnectionFailure: Unable to connect to Cromwell (handshake alert:  unrecognized_name)`.

Since we're in a world where Cromwell will be on `http` for the near future, I'll comment and remove